.github/workflows/review.yaml

```diff
@@ -89,10 +100,10 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v4
 
-      - name: Setup Python ${{ matrix.version }}
+      - name: Setup Python ${{ env.DEFAULT_PYTHON }}
```

Comment:
Fun thing: because the annotations output is really simple code, having a random python version was only breaking the pip cache. Which I noticed because this stage was also installing all the mewbot dependencies for a bit and that takes _forever_.

Reply:
It does indeed! Good change.